flows/parking/atd_parking_data_meters_txn_history.py

```diff
@@ -217,20 +229,21 @@ def update_last_exec_time():
     storage=GitHub(
         repo="cityofaustin/atd-prefect",
         path="flows/parking/atd_parking_data_meters_txn_history.py",
-        ref="production",  # The branch name, staging is not being used here
+        ref="ch-parking-data-secrets",  # The branch name, staging is not being used here
```

Comment:
These branch names in here are probably not ideal, and I like your idea that you shared with me recently about using a value out of the environment / kv store to control this sort of thing. I'm not suggesting that you adopt it here unless you want to and have the bandwidth, but it occured to me that your solution would fit in with this. Am I fitting this all together right in my head?

Reply:
Yeah we should pair on trying to set something up where we can check what branch we're in and just plug that in here? Maybe `production` agents just use `main` and otherwise it's up to you to manually set the branch reference. This gets me all the time!